db/db_impl.cc

```diff
@@ -5544,6 +5544,29 @@ ColumnFamilyHandle* DBImpl::GetColumnFamilyHandle(uint32_t column_family_id) {
   return cf_memtables->GetColumnFamilyHandle();
 }
 
+void DBImpl::GetApproximateMemTableStats(ColumnFamilyHandle* column_family,
+                                         const Range* range,
+                                         int n, uint64_t* counts,
+                                         uint64_t* sizes) {
```

Comment:
Do MyRocks need to get the same of multiple ranges in one function call? If not, I suggest we keep it simple. Just support one range.